client/src/components/overview/productInfo/index.js

```diff
@@ -3,14 +3,18 @@ import ProductRating from './ProductRating';
 import ProductDetails from './ProductDetails';
 import SocialCard from './SocialCard';
 
-const ProductInfo = () => (
+const ProductInfo = ({ currentProduct, selected }) => (
   <div data-testid="productInfo">
-    ProductInfo component
     <section>
-      <ProductRating />
+      <ProductRating currentProduct={currentProduct} />
     </section>
     <section>
-      <ProductDetails />
+      <ProductDetails
+        category={currentProduct.category}
+        name={currentProduct.name}
+        original_price={Number(selected.style.original_price)}
```

Comment:
Why are we converting to a number here only to convert back to a string in ProductDetails? Is it necessary to the conditional rendering logic?

Reply:
Yes, it's for the conditional rendering logic